project/AssembleLWC.scala

```diff
@@ -0,0 +1,172 @@
+package quasar.s3.project
+
+import coursier._
+import java.nio.file.{Path, Files}
+import java.nio.file.StandardCopyOption.REPLACE_EXISTING
+import java.nio.file.StandardOpenOption.{CREATE_NEW, TRUNCATE_EXISTING}
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.collection.JavaConverters._
+import scala.sys.process._
+import java.util.stream.Collectors
+import java.lang.Runtime
+
+import sbt.{Cache => _, MavenRepository => _, Tags => _, Task => _, _}
+import sbt.Keys._
+import scalaz._, Scalaz._
+import Tags.Parallel
+import scalaz.concurrent.Task
+import Task._
+
+object AssembleLWC {
```

Comment:
We'll want this ultimately not in any specific lwc's repo but in a place accessible to all lwc's.

@edmundnoble have you put any thought yet into how we can package up the lwc packaging infrastructure so it'll be available to all lwc's?